react/features/toolbox/components/web/Toolbox.js

```diff
@@ -22,6 +22,7 @@ import {
     getLocalParticipant,
     hasRaisedHand,
     haveParticipantWithScreenSharingFeature,
+    PARTICIPANT_ROLE,
```

Comment:
put the all caps constants at the top please.

Reply:
Fixed.